products/gke.md

```diff
@@ -151,6 +151,8 @@ Clusters with a static GKE version are not enrolled in a release channel. Users
 managing their upgrade strategy in this case. They must still adhere to the Kubernetes version and
 version skew support policy, and use supported GKE versions.
 
+This is the reference used to obtain the release information displayed on this page.
```

Comment:
The sentence is OK, but will move it to a callout (like done on https://endoflife.date/akeneo-pimp) so that it is more visible.

Reply:
Done, see https://deploy-preview-5572--endoflife-date.netlify.app/google-kubernetes-engine. Let me know if the description is OK.

CC @captn3m0 .

Reply:
Good idea LGTM 👍